rust/user-info-fetcher/src/http_error.rs

```diff
@@ -0,0 +1,55 @@
+use std::sync::Arc;
```

Comment:
Could be moved out into some stackable-axum-commons library

Reply:
> Github makes it looks like you are commenting about use std::sync::Arc which is confusing, and I've assumed your comment refers to the whole file

Yes, correct. Goddamnit, github.. :/

Reply:
> I couldn't see a repo or a module by that name.

And yeah, this would be a new thing.

Reply:
Given the new issue I created, can this comment be resolved? Or is it important enough to do this move now?

Reply:
I'd say that's fine.